pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -710,13 +710,8 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			Dependencies: flow.NewTaskIDs(waitUntilGardenerResourceManagerReady, initializeShootClients, ensureShootClusterIdentity, deployKubeScheduler, waitUntilShootNamespacesReady),
 		})
 		deployManagedResourcesForAddons = g.Add(flow.Task{
-			Name: "Deploying managed resources for system components and optional addons",
-			Fn: flow.TaskFn(func(ctx context.Context) error {
-				if isWorkerless {
-					return nil
-				}
-				return botanist.DeployManagedResourceForAddons(ctx)
-			}).RetryUntilTimeout(defaultInterval, defaultTimeout).SkipIf(o.Shoot.HibernationEnabled),
+			Name:         "Deploying managed resources for system components and optional addons",
```

Comment:
Merge this in shoot reconcile flow adaption commit. Also change the commit message.